### PR TITLE
UPSTREAM: 67236: fix azure disk create failure due to sdk upgrade

### PR DESCRIPTION
**Pull request by wongma7**
fix https://bugzilla.redhat.com/show_bug.cgi?id=1613676

### CODE REVIEW

**Comment by wongma7** [comment]
@openshift/sig-storage *

File: vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/azure/azure_blobDiskController.go [hunk 1]
```diff
@@ -488,7 +488,9 @@ func (c *BlobDiskController) createStorageAccount(storageAccountName string, sto
 		glog.V(2).Infof("azureDisk - Creating storage account %s type %s", storageAccountName, string(storageAccountType))
 
 		cp := storage.AccountCreateParameters{
-			Sku:      &storage.Sku{Name: storageAccountType},
```
Comment (gnufied) [inline vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/azure/azure_blobDiskController.go]: heh, something ate the indentation. :-)

**Comment by wongma7** [comment]
/assign @smarterclayton 
please approve, test blocker, ty.

**Comment by knobunc** [comment]
/lgtm
/approve 

Over to you @smarterclayton

**Pull request merged**